config/karma.saucelabs.js

```diff
@@ -119,19 +114,50 @@ module.exports = function(config) {
 
     customLaunchers: sauceLabsBrowsers,
 
-    reporters: ['spec', 'saucelabs'],
+    reporters: ['spec', 'summary', 'saucelabs'],
 
     port: 9876,
 
-    retryLimit: 3,
+    retryLimit: 0,
 
     // concurrency: 10,
 
+    client: {
+      mocha: {
+        timeout: 20000,
+        retries: 3,
+        grep:
+          'Crawler Support|local timestamp|raises error event|handles failures',
+        invert: true
+      }
+    },
+
+    browserConsoleLogOptions: { terminal: false },
```

Comment:
Does this mean we won't get logs from these tests?  That'll make it harder to diagnose failures.  If nothing else, is a comment warranted?

Reply:
Travis has a limit for total number of log lines at 10k. Since we are running 4 browsers now and have more to come (IE11 and mobile browsers), we basically generate a lot more logs, and the summary of test results can be only seen in the raw log. However, it's one more click away and it doesn't display some character correctly. We can still access the console log from saucelabs test result (e.g. https://saucelabs.com/beta/tests/f5ef37f915cb4ae8bc500b9b4ab3ba62).

So I sort of prefer to reduce the log verbosity here. But this is purely a cosmetic reason. Turned console log on now as we are not hitting the limit yet.